### ISSUE TITLE
repeated mails already qualify a "known contact"

### ISSUE THREAD

**Issue by testbird** [deltachat/deltachat-core#89]
Trying to understand how an online shop order confirmation could have made it to appear in DC.

Could it be that a first "account confirmation" email let DC already qualify the sender as a known contact, and thus the second mail "order confirmation" was considered for a chat?

From the FAQ I read that a new address should only be considered as known after it was actively contacted by me or in CC/To by an already known contact.

I am not sure about the best approach, adjusting the FAQ or the known contact qualification?
If repeated mails actually qualify a known contact, then the conditions to prompt/open chat are not so clear cut.

**Comment by r10s** [comment]
Currently, it will only appear in the manually opened "Contact request" (btw. I like the name "E-mail log" that pop up in anothe thread).

I am not yet sure about what is the best approach here, however, at least by default, I do not want to users to be confronted with messages from unknown senders in the chat list. 

Maybe it is a good idea to add an option to show the "e-mail log" in the chatlist (similar to past versions) which, however, is "off" by default.

**Comment by testbird** [comment]
Ok, retried deleting a known address and then a message from that address only appeared in the "Email Log" and was not added to the known contacts. This looks fine to me. If the behavior is also the same when folders are scanned I am still puzzled where the "known addresses" that I actually don't know came from.
So I'm closing issue, it does not seem to be the case.

**Comment by testbird** [comment]
scanned => not rerieved through INBOX watching

**Comment by testbird** [comment]
> I am not yet sure about what is the best approach here, however, at least by default, I do not want users to be confronted with messages from unknown senders in the chat list.

> Maybe it is a good idea to add an option to show the "e-mail log" in the chatlist (similar to past versions) which, however, is "off" by default.

Currently, the user needs to continuously look through the Email Log to catch chat attempts by **unknown** contacts. So the Email Log is required to let new contacts into a chat, and it may seem a little inconsistent to hide the Email Log by default (seems to assume a complete address book or pre-adding address book entries before being reachable by new contacts). => I updated https://github.com/deltachat/deltachat-core/wiki/Use-Cases with a corresponding usabilty option 2.1).

As you saw by my first issues I also found it confusing, and I think seeing the Email Log in the beginning and seeing emails appear (silently) therein could have made it easier to grasp what is happening, and then understand and adjust available options as desired.
I also only saw github comments asking to keep or show the Email Log in the chat or not to hide it by default.
For example in https://github.com/deltachat/deltachat-core/issues/48

So maybe with an explanatory name there is no need to hide the silent "(...) Email Log" at install time anymore.
